views/table.ejs

```diff
@@ -0,0 +1,151 @@
+												<h4 class="modal-title" style="float:left">Table <%= table[i].tableNumber%></h4>
+											</div>
+											<div style="margin:10px; text-align: center;">Table Labels</div>
+											<div id="labelGroup<%=i%>" style="padding:2px">
+											<% for(var j=0; j < table[i].tableLabels.length; j++) { %>
+												
+													<input type="name" name="label-<%=j%>" class="labels<%=j%> form-control" value="<%= table[i].tableLabels[j]%>" name="label" rows="5" required><br>												
+												
+											<%}%> 
+											</div>	
+											<% if(table[i].tableLabels.length ==0){%>
+												<div id="noLabels<%=i%>" style="margin:10px; text-align: center;">-No labels-</div>
+											<%}%>
+											<div data-table="<%=i%>" class="addLabel btn btn-secondary btn-sm">Add label</div>
+											<div class="modal-body" >
+												<div style="padding:2px;text-align: center;">Seat Allocation</div>
+												<% for(var x=0; x < table[i].seatLabels.length; x++) { %>
+													Seat <%= x+1%> :<textarea type="name" class="form-control" name="seat-<%=x%>" rows="1" required><%= table[i].seatLabels[x]%></textarea> <br>
+												<% }%> 
+											</div>
+											<div class="modal-footer">		
+												<button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>							
+												<button type="submit" class="btn btn-success">Confirm</button>
+											</div>
+										</div>
+									</div>
+								</div>			    
+							</div>		
+						</form>		
+						<!-- Delete Model-->
+						<div class="row">
+								<form action="/table/delete/<%=event_id%>/<%=table[i].tableNumber%>" method="post" id='form'>
+									<div id="deleteModal<%=i%>" class="modal fade" role="dialog">
+										<div class="modal-dialog">
+											<div class="modal-content">
+												<div class="modal-header">
+													<button type="button"style="float:left;margin:0;" class="close" data-dismiss="modal">&times;</button>
+													<h4 class="modal-title" style="float:left">Delete Table</h4>
+												</div>
+												<div style="padding:10px;">
+													<label >Are you sure you want to <b>delete table <%= table[i].tableNumber%>?</b></label>
+												</div>
+												
+												<div class="modal-footer">		
+													<button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>							
+													<button type="submit" class="btn btn-success">Confirm</button>
+												</div>
+											</div>
+										</div>
+									</div>		
+								</form>	
+							</div>
+
+
+					</div>
+				</div>	
+				<%}%>
+			</div>			
+			
+		
+		</div>
+			<%}else{%>
+			<div style="text-align:center">-No tables allocated-</div>
+			<%}%>
+			<div class="row">
+				<button type="button" style="margin-bottom:10px;"class="btn btn-warning" data-toggle="modal" data-target="#tableModal">Create Table</button>				
```

Comment:
This is my personal preference so ignore if theres a reason for it: There's likely to be multiple tables and they could be quite large because of this the create table button ends up a long way down the page, so might be better suited above the tables